setup.cfg

```diff
@@ -12,7 +12,9 @@ classifiers =
     Intended Audience :: Science/Research
     License :: OSI Approved :: BSD License
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+	Programming Language :: Python :: 3.9
+	Programming Language :: Python :: 3.10
```

Comment:
Why are the `3.x` lines indented differently?

Reply:
Thanks! VS Code made it look the same...seems to be rendering tabs weirdly. Advice welcome :)